scripts/docker-setup.sh

```diff
@@ -2,7 +2,7 @@
 if ! [ -d "local-setup" ]
 then
     git clone https://github.com/interlay/parachain-launch/
-    cd parachain-launch && yarn install
+    cd parachain-launch && git checkout 1.1.0-20220613 && yarn install
```

Comment:
I feel like this version could get lost easily, I wonder if we should set it from an env config or something?

Reply:
> I feel like this version could get lost easily, I wonder if we should set it from an env config or something?

I haven't quite figured out how to best do that yet. It feels like, either way, there will be one more place to update the commit hash or tag.

I think my favorite way would be if I could have it in package.json to then be extracted and used in the script... somehow